### DISCUSSION TITLE
httpx is not scanning all the domains in the list.

### DISCUSSION THREAD

**Discussion by om-prabhat** [projectdiscovery/httpx#1735] [Q&A]
![Screenshot 2024-05-30 221804](https://github.com/projectdiscovery/httpx/assets/144362728/283f6bb4-6a9a-419c-a3cb-36738c9eb384)

**Comment by Bundy01** [comment]
Hi,

I encountered the same issue when testing httpx.
Regards.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
